src/plugins/theme/spacing.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Spacing - used in various classes such as margin and padding; individual values taken from Responsive Spacing map
+ */
+module.exports = function () {
+  return {
+    0: '0',
+    '01em': '0.1em',
+    '02em': '0.2em',
+    '03em': '0.3em',
+    '04em': '0.4em',
+    '05em': '0.5em',
+    '06em': '0.6em',
+    '07em': '0.7em',
+    '08em': '0.8em',
+    '09em': '0.9em',
+    '1em': '1em',
```

Comment:
Added these as I often find it useful to have "em" spacing for the small end of things, e.g, spacing between link and link icon.